makefile

```diff
@@ -1,7 +1,3 @@
-bundle:
-	echo "// Copyright 2021 Deno Land Inc. All rights reserved. MIT license.\n// deno-lint-ignore-file\n// deno-fmt-ignore-file" > src/runtime.bundle.js
-	deno bundle src/runtime.js >> src/runtime.bundle.js
-
```

Comment:
I wonder if we can remove this file entirely.

nit: we're not using any unstable APIs, so the `test` step can remove the `--unstable` flag

Reply:
> nit: we're not using any unstable APIs, so the test step can remove the --unstable flag

I am just keeping it so that in the eventuality that we add a unstable api in a future version of `deployctl`, people upgrading with `deployctl upgrade` won't get a broken experience.

Reply:
Let's keep the file. `make test` is easier to type than `deno test -A --unstable` :wink: